client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/Forms.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+import useFormStyles from 'styles/formStyles';
+
+import FamilyMembersForm from './FamilyMembers/FamilyMembersForm';
+import ManualContactsForm from './ManualContactsForm/ManualContactsForm';
+import GroupedInvestigationsTab from './GroupedInvestigations/GroupedInvestigationsTab';
+
+const Forms = (props: Props) => {
+    const { currentTab,  groupedInvestigationContacts, setGroupedInvestigationContacts} = props;
+    const formClasses = useFormStyles();
+
+    const contactFormTabs = [
```

Comment:
I've separated the pages so that ContactsTabs is the tabs themselves (on the right) and the "Forms" are the side on the left that's changing according to the right , so that there exists a hierarchy between them

Reply:
👍🏼